test/rules/completed-docs/accessors/test.ts.lint

```diff
@@ -45,7 +45,8 @@ export class CommentsOnGetOnly {
      */
     get myAccessor(): string { return this.myField; }
 
-    set myAccessor(value: string) { this.myField = value; } // Comments for the setter are inherited from the getter.
+    set myAccessor(value: string) { this.myField = value; }
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~        [Documentation must exist for properties.]
```

Comment:
I would actually avoid adding a failure here. The property is documented. Why duplicate the comment?